crates/gfx/src/renderer.rs

```diff
@@ -267,7 +269,7 @@ impl<UIFn, Data> Renderer<UIFn, Data> {
 
         let light_uniform_data = PointLightUniform {
             position: [5.0, 5.0, 5.0],
-            _padding: 0,
+            is_visible: 0, // Invisible by default.
```

Comment:
Yeah, it's because when passing data to the graphics card (in the shaders) it doesn't accept bools so you need to use a number instead. We considered using a byte (`u8`), but since the padding was already four bytes we thought that we might as well use all of it (since it'd need to be filled out by padding anyways).